multiqc/modules/vcftools/vcftools.py

```diff
@@ -0,0 +1,14 @@
+from multiqc.modules.base_module import BaseMultiqcModule
+from .relatedness import Relatedness2Mixin
+
+
+class MultiqcModule(BaseMultiqcModule, Relatedness2Mixin):
+    def __init__(self):
+        super(MultiqcModule, self).__init__(
+            name='VCFTools',
+            anchor='vcftools',
+            href='https://vcftools.github.io',
+            info='is a program for working with and reporting on VCF files'
+        )
+
+        self.parse_relatedness2()
```

Comment:
It's good to get this function to return the number of samples parsed (either input files or samples - up to you which). So the main thing is that it's 0 if nothing was found. Then, we can run this afterwards:

```python
# Exit if we didn't find anything
if sum(n.values()) == 0:
        log.debug("Could not find any reports in {}".format(config.analysis_dir))
        raise UserWarning
```